packages/x-components/src/x-modules/popular-searches/components/popular-search.vue

```diff
@@ -21,8 +22,8 @@
   import Vue from 'vue';
   import { Component, Prop } from 'vue-property-decorator';
   import BaseSuggestion from '../../../components/suggestions/base-suggestion.vue';
-  import { xComponentMixin } from '../../../components/x-component.mixin';
-  import { XEventsTypes } from '../../../wiring/events.types';
+  import { xComponentMixin } from '../../../components';
+  import { XEventsTypes } from '../../../wiring';
```

Comment:
Avoid import from barrels even if webstorm suggests you to shorten the import

Reply:
Ok :ok_hand: